src/Applicator/QueryBuilderApplicator.php

```diff
@@ -0,0 +1,57 @@
+<?php declare(strict_types=1);
+
+namespace Lmc\ApiFilter\Applicator;
+
+use Doctrine\ORM\QueryBuilder;
+use Lmc\ApiFilter\Entity\Filterable;
+use Lmc\ApiFilter\Filter\FilterInterface;
+use Lmc\ApiFilter\Filter\FilterWithOperator;
+
+class QueryBuilderApplicator extends AbstractApplicator
+{
+    public function supports(Filterable $filterable): bool
+    {
+        return $filterable->getValue() instanceof QueryBuilder;
```

Comment:
Tohle bude fungovat, když nebudu mít doctrine? Vím, že někde ten if na `class_exists` už je, ale raději bych ho dal i sem.
`class_exists('Doctrine\ORM\QueryBuilder') && $filterable->getValue() instanceof QueryBuilder;`

Reply:
Yes it won't work without `doctrine/orm` but you should not get an instance of this class unless you have that library in your project.